pcl_config.h.in

```diff
@@ -100,3 +100,5 @@
 
 #cmakedefine HAVE_QVTK 1
 
+#cmakedefine BUILD_visualization
```

Comment:
I feel like `BUILD_visualization` might not send a clear message to a person using PCL. Maybe it would be better to introduce a separate preprocessor symbol, like `PCL_VISUALIZATION_AVAILABLE` or similar, and use that in the *.h and *.hpp files (this should then be set to true in visualization/CMakeLists.txt if the visualization module is indeed built).

Reply:
Yes, a better name would be nice. I tried your suggestion, but couldn't make it work. The problem is that `pcl_config.h.in` is generated before the PCL module CMake files are processed (generating it afterwards does not work). So I'm setting in in the main `CMakeLists.txt` now.

In `outofcore/CMakeLists.txt`, I still use `BUILD_visualization`. We could switch that to `PCL_VISUALIZATION_AVAILABLE` if you want.